queue_services/entity-filer/src/entity_filer/filing_processors/__init__.py

```diff
@@ -48,20 +48,72 @@ def update_address(address: Address, new_info: dict):
     return address
 
 
-def update_director(director: Director, new_info: dict):
+def create_director(director_info: dict):
```

Comment:
Should this create the director in both old & new structures until after data migration and migrating the code to using parties?

Reply:
The director is also created as before in the processor. Since I'll be taking out the director table in my next PR and removing that code anyways I didn't think it was worth it to refactor adding the director into this method :P